<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMyFindWindow</class>
 <widget class="QDialog" name="CMyFindWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/u=373127414,4048740494&amp;fm=26&amp;gp=0.jpg</normaloff>:/new/prefix1/u=373127414,4048740494&amp;fm=26&amp;gp=0.jpg</iconset>
  </property>
  <widget class="QPushButton" name="btn_seletStuId">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_subName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 课程 ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stu">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_seletSubName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>学生ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>分数</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_stuName">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;查询选课信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 学生 ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_seletsubName">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>统计</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 课程 名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;查询学生信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_subId">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_stuName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 课程 ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;查询课程信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_stuId">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 课程 名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 学生姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_seletSubId">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_subName">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请输入 学生 ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_seletsubId">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_seletStu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_stu</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>stuInfoFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stuName</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>stuIdFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sub</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>subInfoFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_subName</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>subIdFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_seletsubId</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>selSubInfoFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_seletStuId</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>selStuInfoFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_seletsubName</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMyFindWindow</receiver>
   <slot>selSubFindSlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stuInfoFindSlots(bool)</slot>
  <slot>stuIdFindSlots(bool)</slot>
  <slot>subInfoFindSlots(bool)</slot>
  <slot>subIdFindSlots(bool)</slot>
  <slot>selSubInfoFindSlots(bool)</slot>
  <slot>selStuInfoFindSlots(bool)</slot>
  <slot>selSubFindSlots(bool)</slot>
 </slots>
</ui>
